package.json

```diff
@@ -16,7 +16,8 @@
     "bump-version": "npm --no-git-tag-version version ",
     "preversion": "git-branch-is develop && yarn test",
     "postversion": "yarn run docs",
-    "make-release": "git-branch-is develop && git flow release start $npm_package_version && git flow release finish $npm_package_version"
+    "make-release": "git-branch-is develop && git flow release start $npm_package_version && git flow release finish $npm_package_version",
+    "coveralls": "cat ./coverage/lcov.info | coveralls"
```

Comment:
This `coveralls` can be deleted since coveralls action is taking care of uploading the coverage file to coveralls.io